### DISCUSSION TITLE
minecraft server version

### DISCUSSION THREAD

**Discussion by Nullcomputer** [AlexProgrammerDE/balena-minecraft-server#103] [Q&A] [answered]
how do u change the version for the minecraft server

**Comment by AlexProgrammerDE** [comment, marked as answer]
You follow the connect to files section of the readme and then replace your jar file with the JAR file of your choice. You should also consider setting the `JAR_FILE` variable if your jar has a different name. It isn't sadly as easy as clicking a single button, but it will work.
